lib/Extractor/KLEEBuilder.cpp

```diff
@@ -934,8 +963,15 @@ CandidateExpr souper::GetCandidateExprForReplacement(
   ExprBuilder EB(CE.Arrays, CE.ArrayVars);
   // Build LHS
   ref<Expr> LHS = EB.get(Mapping.LHS);
-  // Build PCs
   ref<Expr> Ante = klee::ConstantExpr::alloc(1, 1);
+  // Build KnownBits Map
+  for (const auto &I : CE.ArrayVars) {
+    if (I && (I->KnownZeros.getZExtValue() || I->KnownOnes.getZExtValue())) {
```

Comment:
again do not do this, it does not work when the apint doesn't fit into a uint64_t